docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
       - "--certificatesResolvers.le.acme.httpChallenge=true"
       - "--certificatesResolvers.le.acme.httpChallenge.entryPoint=web"
       - "--experimental.http3=true"
-      - "--entrypoints.web-secure.enablehttp3=true"
+      - "--entrypoints.web-secure.http3"
```

Comment:
My block looks like

```
experimental:
  http3: true

entryPoints:
  web:
    address: ":80"
  web-secure:
    address: ":443"
    http3: {}
  mqtt:
    address: ":8883"
```

So maybe make that a `--entrypoints.web-secure.http3={}`.

Reply:
But https://doc.traefik.io/traefik/routing/entrypoints/#http3 says `--entrypoints.name.http3` for the cli, so maybe it's ok. 😁

Reply:
thanks. I'll it then :)